src/data/queries/tests/__snapshots__/vamcSystemVaPolice.test.tsx.snap

```diff
@@ -113,6 +113,12 @@ exports[`VamcSystemVaPolice formatData outputs formatted data 1`] = `
   ],
   "moderationState": "published",
   "path": "/central-arkansas-health-care/va-police",
+  "policeOverview": {
+    "html": "<p>VA police officers help make VA medical centers and other VA health facilities safe for Veterans and their family members.</p>
```

Comment:
Not sure why the dark red highlighting here. I tried to change this by removing the line break, but then it failed to match the mock, so I left it like this.